apps/omg/lib/omg/state.ex

```diff
@@ -84,27 +86,14 @@ defmodule OMG.State do
   ### Server
 
   @doc """
-  Start processing state using the database entries
+  Initializes the state. UTXO set is not loaded now.
```

Comment:
don't have a better place to put the comment: we need to do sth about the utxo-based metrics calculations :(. Out of ideas for now

Reply:
I've created an isse for it #1148